OMCompiler/Compiler/BackEnd/SymbolicJacobian.mo

```diff
@@ -2394,6 +2394,29 @@ algorithm
   end match;
 end getJacobianMatrixbyName;
 
+public function updateJacobianDependencies
```

Comment:
Convenience function, not jet in use but since i already made it i wanted to push it. It is somewhat related to this PR anyway.